README.md

```diff
@@ -54,8 +54,22 @@ https://www.digitalocean.com/community/articles/geddy-js-a-no-brainer-mvc-node-j
 
 #### License
 
-Apache License, Version 2
+Copyright 2017. Geddy - Web-app development framework
```

Comment:
@mde, please realize that deleting complete sentences of other peoples comments is an act of barbarism. It's called "censorship", and its far more "off-limits" than using the word "sh*t".

If you have such a big problem with a word ("sh*t"'s off-limit just tries to describe the off-limit condition of the project), then just use "*" to edit a comment.